spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -354,7 +364,9 @@
     end
   end
 
-  xit "cannot edit a user's profile" do
+  it "cannot edit a user's profile" do
+    visit root_path
+    expect(page).to_not have_content('Edit Profile')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.